stories/annotations.stories.tsx

```diff
@@ -0,0 +1,192 @@
+import { Meta } from '@storybook/react';
+import React from 'react';
+
+import { Axis, LineSeries, Plot, Legend, Annotation } from '../src';
+
+import { DEFAULT_CONFIG } from './utils';
+
+const shapes = ['circle', 'triangle', 'diamond', 'square'];
+
+export default {
+  title: 'Docs/Annotations',
+  decorators: [
+    (Story) => (
+      <Plot
+        {...DEFAULT_CONFIG}
+        seriesViewportStyle={{ stroke: 'black' }}
+        margin={{
+          bottom: 100,
+          left: 100,
+          top: 100,
+          right: 200,
+        }}
+      >
+        <Legend position="embedded" />
+
+        <LineSeries
+          data={data}
+          xAxis="x"
+          yAxis="y"
+          label="Label line series"
+          markerShape="pentagon"
+          markerStyle={{ fill: 'green' }}
+          displayMarker
+        />
+
+        <Annotations>
+          <Story />
+        </Annotations>
+
+        <Axis id="x" position="bottom" label="X" />
+        <Axis id="y" position="left" label="Y" />
+      </Plot>
+    ),
+  ],
+} as Meta;
+
+const { Annotations } = Annotation;
+
+const data = [
+  {
+    x: 0,
+    y: 10,
+  },
+  {
+    x: 1,
+    y: 12,
+  },
+  {
+    x: 2,
+    y: 14,
+  },
+  {
+    x: 3,
+    y: 16,
+  },
+  {
+    x: 4,
+    y: 18,
+  },
+];
+
+export function AnnotationRectangleStories() {
```

Comment:
Can you remove the stories suffix from all exported stories?

They end up in the story name

Reply:
i have updated the stories, can you say to me if its good for you ?